src/main/java/com/fabriccommunity/spookytime/models/TreasureChestModel.java

```diff
@@ -0,0 +1,43 @@
+package com.fabriccommunity.spookytime.models;
```

Comment:
`com.fabriccommunity.spookytime.client.models` would be a better place as all existing client classes are in the client package.